web/ui/react-app/src/pages/tsdbStatus/TSDBStatus.tsx

```diff
@@ -12,21 +12,53 @@ interface Stats {
 }
 
 export interface TSDBMap {
+  numSeries: number;
+  chunkCount: number;
+  minTime: number;
+  maxTime: number;
   seriesCountByMetricName: Stats[];
   labelValueCountByLabelName: Stats[];
   memoryInBytesByLabelName: Stats[];
   seriesCountByLabelValuePair: Stats[];
 }
 
 export const TSDBStatusContent: FC<TSDBMap> = ({
+  numSeries,
+  chunkCount,
+  minTime,
+  maxTime,
   labelValueCountByLabelName,
   seriesCountByMetricName,
   memoryInBytesByLabelName,
   seriesCountByLabelValuePair,
 }) => {
+  const unixToTime = (unix: number): string => new Date(unix).toISOString();
+  const stats = [
+    { header: 'Number of Series', value: numSeries },
+    { header: 'Number of Chunks', value: chunkCount },
+    { header: 'Current Min Time', value: `${unixToTime(minTime)} (${minTime})` },
+    { header: 'Current Max Time', value: `${unixToTime(maxTime)} (${maxTime})` },
+  ];
   return (
     <div>
       <h2>TSDB Status</h2>
+      <h3 className="p-2">Head Stats</h3>
+      <Table bordered size="sm" striped>
```

Comment:
All other tables are embedded in a `className="p-2"` div, causing them to have some padding around them, but this one doesn't, so it stands out more to the sides. Could you make it consistent?